### DISCUSSION TITLE
Does capitalization matter in the dataset?

### DISCUSSION THREAD

**Discussion by Weroxig** [erew123/alltalk_tts#125] [Q&A] [answered]
I ran the finetune.py on my dataset, but looking into the metadata_train.csv and metadata_eval.csv, capitalisation is ignored. When prompting the model, does capitalization matter at all?

Also, can I just change the .csv files after running whisper to modify things in case whisper got it wrong? Is that how it works?

**Comment by erew123** [comment, marked as answer]
Hi @Weroxig 

**I ran the finetune.py on my dataset, but looking into the metadata_train.csv and metadata_eval.csv, capitalisation is ignored. When prompting the model, does capitalization matter at all?**
It doesn't matter in any way that I am aware of.

**Also, can I just change the .csv files after running whisper to modify things in case whisper got it wrong? Is that how it works?**
Yes you can change/edit these before moving to the training step.

Thanks
